src/components/Table/hoc/withTableSettings/TableColumnSetup/DndTreeSelect.tsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+
+import {Grip} from '@gravity-ui/icons';
+import {DragDropContext, Draggable, Droppable} from 'react-beautiful-dnd';
+import type {OnDragEndResponder} from 'react-beautiful-dnd';
+
+import {useUniqId} from '../../../../../hooks';
+import {Icon} from '../../../../Icon';
+import {TreeSelect} from '../../../../TreeSelect/TreeSelect';
+import {TreeSelectItem} from '../../../../TreeSelect/TreeSelectItem';
+import type {TreeSelectItemProps} from '../../../../TreeSelect/TreeSelectItem';
+import type {RenderContainerType, RenderItem, TreeSelectProps} from '../../../../TreeSelect/types';
+import {ListContainerView} from '../../../../useList';
+import {reorderArray} from '../../../../useList/__stories__/utils/reorderArray';
```

Comment:
you don't need to import utility code from stories `__stories__`.
specify it logic on component level

Reply:
88ffdc67918ed5209a640f783df5d4bc00641d7c